akka-actor/src/main/scala/akka/serialization/Serialization.scala

```diff
@@ -433,6 +437,14 @@ class Serialization(val system: ExtendedActorSystem) extends Extension {
     }
   }
 
+  private def warnUnexpectedNonAkkaSerializer(clazz: Class[_], ser: Serializer): Boolean = {
+    if (clazz.getName.startsWith("akka.") && !ser.getClass.getName.startsWith("akka.")) {
+      log.warning("Using serializer [{}] for message [{}]. Note that this is not a serializer " +
```

Comment:
should we be more strong and say that we do not recommend doing this in the message? 

This will get very spamy for users who think they have a genuine reason for doing this (are there any??), should we turn it off with a config switch?

Reply:
Yes, I can rephrase.

It’s once per message class so not too bad.

Reply:
rephrased